src/services/enum-utilities.ts

```diff
@@ -251,7 +251,6 @@ export default class EnumUtilities {
       case FilingTypes.DISSOLUTION: return FilingNames.DISSOLUTION
       case FilingTypes.DISSOLVED: return FilingNames.DISSOLVED
       case FilingTypes.INCORPORATION_APPLICATION: return FilingNames.INCORPORATION_APPLICATION
-      case FilingTypes.INVOLUNTARY_DISSOLUTION: return FilingNames.INVOLUNTARY_DISSOLUTION
```

Comment:
There is no such thing as a filing type of "involuntary dissolution". Instead, the type is "dissolution" and the subtype is "involuntary".

Ditto for voluntary dissolution below.

Reply:
Ref: https://github.com/bcgov/business-schemas/blob/f70aef8ff39ac6ef2cd6194362aa10ac14260197/src/registry_schemas/schemas/filing.json#L44